File: max.js
const max = () => {
    const userInput = prompt('Enter the array');

    let array = userInput.split(",").map((item => Number(item.trim())));

    let max = 0;
    for(let i = 0;i < 3;i++)
    {
        if(array[i] > max)
        {
            max = array[i];
        }
    }

    console.log(`Maximum number is ${max}`);
}

max();

File: max.java
import java.util.Scanner;
class A
{
    int maxnum(int a[], int max)
    {
        for(int i = 0;i < a.length;i++)
        {
            if(a[i] > max)
            {
                max = a[i];
            }
        }
        return max;
    }
    public static void main(String args[])
    {
        A ss2 = new A();
        Scanner ss = new Scanner(System.in);
        int a[] = new int[3];
        System.out.println("Enter the input");
        for(int i = 0;i < a.length;i++)
        {
            a[i] = ss.nextInt();
        }
        int max = 0;
        System.out.println("Maximun number is\s" + ss2.maxnum(a, max));
        ss.close();
    }
}